GDJS/GDJS/IDE/ExporterHelper.cpp

```diff
@@ -287,6 +290,22 @@ bool ExporterHelper::ExportCordovaFiles(const gd::Project &project,
             "\t</plugin>");
   }
 
+  gd::String pluginsList = "";
+
+  for(gd::String const extensionName : project.GetUsedExtensions()) {
+    std::shared_ptr<gd::PlatformExtension> extension = project.GetCurrentPlatform().GetExtension(extensionName);
+    if (std::shared_ptr<gdjs::CordovaExtension> cordovaExtension = std::dynamic_pointer_cast<gdjs::CordovaExtension>(extension)) { // If a downcast to CordovaExtension is possible...
+      for(gd::String const cordovaPluginName : cordovaExtension->GetCordovaPluginList()) {
+        str += "<plugin name=\"" + cordovaPluginName + "\">\n"; 
+      }
+    }
+  }
+
+  str.FindAndReplace(
```

Comment:
This is the part I need help with. Seems pretty simple to me but after compiling and restarting the dev server and electron client, the export still succeeded but didn't replace `<!-- GDJS_CORDOVA_PLUGINS -->`. 
This might be an issue with my workspace but I need others to test to be sure / for them to correct any apparent newbie mistake I could have made (not very familiar with cpp).

Reply:
Mmm that should work. Is the comment not replaced at all? This means that your code did not run (otherwise it would be at least replaced by an empty string).

Have you recompiled GDevelop.js? If yes it should work after reloading GDevelop, because GDevelop.js copy itself to newIDE.